src/dom.ts

```diff
@@ -20,6 +20,50 @@ for (const param in tagWrap) {
 	tw.post = '</' + tw.reverse().join('></') + '>';
 }
 
+function validateClass(token: string): void {
+	if (!token || token === '') {
```

Comment:
Actually...  scratch that.  We should _only_ be checking `token === ''`.

The spec is ambiguous about what to do for null or undefined, but it appears that what actual implementations do is coerce to string first.  This causes `null` to result in `'null'` and `undefined` to result in `'undefined'`, etc.

As silly as that seems, if we want to be consistent with the behavior of native `classList`, we should consider coercing to string and returning the coerced value back to the caller.  (We should still keep the `string` type on the parameter too, but this will cover us for `null`).